packages/cashier/src/stores/account-transfer-store.ts

```diff
@@ -514,19 +514,32 @@ export default class AccountTransferStore {
         const accounts = this.accounts_list;
         const selected_from = accounts.find(account => account.value === target.value);
 
+        const isMt5BetweenCFDsTransfer =
```

Comment:
Make a config file and map its values instead of repeating the same thing.
`const platforms = ['is_mt', 'is_ctrader', 'is_dxtrade'];
platforms.map(platform => (selected_from?.platform && this.selected_to.is_mt)) etc
`

Reply:
@hirad-deriv 
sure let me try to do this